syz-manager/manager.go

```diff
@@ -540,6 +552,56 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		procs = 1
 	}
 
+	var rawDlkms []byte
+	waitForOutput := func(outc <-chan []byte) {
+		timer := time.NewTimer(10 * time.Second).C
+		for {
+			select {
+			case out, ok := <-outc:
+				if !ok {
+					return
+				}
+				rawDlkms = append(rawDlkms, out...)
+			case <-timer:
+				return
+			}
+		}
+	}
+
+	// Find addresses of DLKMs
+	dlkmC, _, _ := inst.Run(time.Hour, mgr.vmStop, "cat /proc/modules")
```

Comment:
/proc/modules is linuxism. syz-manager is OS independent and supports 7 operating systems.